util/log.py

```diff
@@ -88,6 +88,11 @@ def config(verbosity=0, logfile=None):
     #   logging.getLogger("dxtbx").setLevel(logging.DEBUG)
     console.setLevel(loglevel)
 
+    # Enable print of messages from warnings.warn to console
+    py_warnings_logger = logging.getLogger("py.warnings")
+    py_warnings_logger.addHandler(console)
+    logging.captureWarnings(True)
+
```

Comment:
This is actually quite intrusive and suggests that what you want to use is `UserWarning` instead of using `DeprecationWarning` and changing the behaviour of all warnings.

Reply:
Just for completeness: [PEP 565 suggests `FutureWarning`](https://www.python.org/dev/peps/pep-0565/) over `UserWarning`. I suppose they behave identically.

Reply:
Will remove and change the message instance over to `logger.warn` and away from `warnings` as this should be pure user land stuff